cmds/grpc-backend/main.go

```diff
@@ -129,7 +130,7 @@ func RunGRPCServer(ctx context.Context, address string) error {
 			KeyResolver:       keyResolver,
 			KeyRefreshTimeout: *keyRefreshTimeout,
 			RequiredScopes:    dssServer.AuthScopes(),
-			RequiredAudience:  *jwtAudience,
+			RequiredAudiences: strings.Split(*jwtAudiences, ","),
```

Comment:
nit: Seems like this is better named "AcceptableAudiences" (mirroring the name of the flag) since no one audience is required unless there is exactly one audience